sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -206,7 +206,7 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
 
 object PROCESS_TABLES extends QueryTest with SQLTestUtils {
   // Tests the latest version of every release line.
-  val testingVersions = Seq("2.1.3", "2.2.2", "2.3.2")
+  val testingVersions = Seq("2.1.3", "2.2.2", "2.3.2", "2.4.0")
```

Comment:
when will we drop 2.1 officially?

Reply:
2.1 should be removed actually. At least according to our default 18-month support rule, it ended support in June 2018, and I see no desire for a 2.1.4 release. 2.2 will end in Jan 2019 according to the same guideline. It's not a hard rule, but probably the right guidance to follow.

Certainly, as this test takes a while and can be flaky I'd like to keep it to 3 versions.